From 625f79cf4997be76a6fc7cee66c75a7bda584613 Mon Sep 17 00:00:00 2001
From: nhannguyen <nhan.nguyen.yb@renesas.com>
Date: Mon, 23 Apr 2018 15:10:49 +0700
Subject: [PATCH 462/504] ARM: dts: iwg21m: Disable EthAVB to enable VIN0 by
 default

This is of iWave u-boot workaround.

In RZ/G1H platform, VIN0 and EthAVD are pin-multiplexed. When SW0 is ON,
VIN0 is selected, otherwise EthAVB is. This patch disables EthAVB and
make VIN0 as the default selection.

Signed-off-by: nhannguyen <nhan.nguyen.yb@renesas.com>
Signed-off-by: Duy Dang <duy.dang.yb@renesas.com>
Signed-off-by: Hao Bui <hao.bui.yg@renesas.com>
---
 arch/arm/boot/dts/r8a7742-iwg21m.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/r8a7742-iwg21m.dts b/arch/arm/boot/dts/r8a7742-iwg21m.dts
index 6432a7e..b82f6fc 100644
--- a/arch/arm/boot/dts/r8a7742-iwg21m.dts
+++ b/arch/arm/boot/dts/r8a7742-iwg21m.dts
@@ -227,7 +227,7 @@
 
 	phy-handle = <&phy0>;
 	renesas,no-ether-link;
-	status = "okay";
+	status = "disabled";
 
 	phy0: ethernet-phy@0 {
 		reg = <3>;
-- 
2.7.4

